src/Robo/Common/Executor.php

```diff
@@ -223,6 +223,8 @@ public function checkUrl($url) {
         return TRUE;
       }
       else {
+        $this->logger->info("Response code: " . $res->getStatusCode());
```

Comment:
Since we ping urls every second or so for some operations (waiting for Drush server to spin up, for instance), doesn't printing the status code and body every time seem awfully verbose? Could this be a debug statement instead (so you'd see it with `blt -vvv`?

Reply:
Yeah, I agree.